app/assets/javascripts/select-kit/addon/components/category-chooser.js

```diff
@@ -147,6 +146,7 @@ export default ComboBoxComponent.extend({
   },
 
   _matchCategory(filter, categoryName) {
-    return this._normalize(categoryName).indexOf(filter) > -1;
+    const regex = RegExp(filter, "i");
```

Comment:
This feels like a hack. Why not just normalize the filter?

Reply:
how is that a hack ?

Reply:
Ok I pushed a version applying my normalize function to the filter

Reply:
Well, it's not if we want regex search for categories, but if that's the case then, the commit is misnamed.